### PR TITLE
Update VeSync Integration page

### PR DESCRIPTION
**Pull request by webdjoe**
**Description:**
Update the documentation for the functionality of the VeSync Integration

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/24953

## Checklist:

- [X ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9779"></a>

### CODE REVIEW

File: source/_components/vesync.markdown [hunk 1]
```diff
@@ -10,27 +10,52 @@ footer: true
 logo: vesync.png
 ha_category:
   - Switch
+  - Fan
 ha_release: 0.66
 redirect_from:
  - /components/switch.vesync/
+ - /components/fan.vesync/
```
Comment (klaasnicolaas) [inline source/_components/vesync.markdown]: Please remove line 17

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/vesync.markdown (hunk 1 above)
Comment (webdjoe) [reply source/_components/vesync.markdown]: Line removed, thanks for the catch

File: source/_components/vesync.markdown [hunk 2]
```diff
@@ -10,27 +10,51 @@ footer: true
...
 
-VeSync outlets are low-cost wifi smart plugs that offer energy monitoring and work with popular voice assistants.
+The devices must be added to the VeSync App before they can be discovered by this component.
+
+The following platforms are supported:
+
+- **switch**
+- **fan**
+
+## {% linkable_title Supported Devices %}
```
Comment (frenck) [inline source/_components/vesync.markdown]:
```suggestion
## Supported Devices
```

Please remove all `linkable_title`'s

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by webdjoe** [comment]
Can you hold off on the approval, I'm working on the tests for the integration and adding another supported device?

**Comment by klaasnicolaas** [comment]
Added WIP to the PR name and you have to resolve a merge conflict 😉

File: source/_components/vesync.markdown [hunk 3]
```diff
@@ -36,13 +54,57 @@ password:
   description: Password needed to log in to VeSync.
   required: true
   type: string
+time_zone:
```
Comment (MartinHjelmare) [inline source/_components/vesync.markdown]: This option was removed, right?

Comment (webdjoe) [reply source/_components/vesync.markdown]: Forgot to push, all good to go

**Review by frenck — APPROVED** [review]
Thanks, @webdjoe! 👍

**Pull request merged**